2-projects/1-todo-list/script.js

```diff
@@ -17,9 +61,27 @@ function addNewTodo(event) {
   // The code below prevents the page from refreshing when we click the 'Add Todo' button.
   event.preventDefault();
   // Write your code here... and remember to reset the input field to be blank after creating a todo!
+  let todoInput = document.getElementById("todoInput");
+  let inputValue = todoInput.value;
+  todos.push({ task: `${inputValue}`, completed: false });
+  populateTodoList(todos);
```

Comment:
This works, but it will be good to look into rendering. Can you explain what happens when you call `populateTodoList` function from here and how it could be improved?